tests/Database/SelectFields/InterfaceTests/LikableInterfaceType.php

```diff
@@ -24,7 +33,12 @@ public function fields(): array
             ],
             'title' => [
                 'type' => Type::nonNull(Type::string()),
-                'alias' => 'title',
+            ],
+            'likes' => [
+                'type' => Type::listOf(GraphQL::type('Like')),
+                'query' => function (array $args, MorphMany $query) {
+                    return $query->whereRaw('1=1');
```

Comment:
This code is never executed => is this expected?

Reply:
It is now resolved, I added a new test